tests/pytests/test_tags.py

```diff
@@ -160,3 +160,21 @@ def testIssue1305(env):
     res = env.cmd('ft.search', 'myIdx', '~@title:{wor} ~@title:{hell}', 'WITHSCORES')[1:]
     res = {res[i]:res[i + 1: i + 3] for i in range(0, len(res), 3)}
     env.assertEqual(res, expectedRes)
+
+def testTagCaseSensitive(env):
+    env.expect('FT.CREATE idx1 SCHEMA t TAG').ok()
```

Comment:
We currently cannot differentiate between a tag without a SEPARATOR, which implicitly gets the default separator,
and a tag with an explicit SEPARATOR (that could define the same separator as the default separator)
But we would need to be able to differentiate for pr #2133  (item (2) in this [comment](https://github.com/RediSearch/RediSearch/pull/2133#pullrequestreview-709308849))
I mention this, although it is not part of this PR, since it will affect this test (`idx1` will not split the tag, but it does split it now),
Or we can change this test to avoid this in this PR to not depend on the implicit separator.

Reply:
It won't affect this test b/c it is on hashes which maintain the original default PR.